src/libcrun/handlers/wasmedge.c

```diff
@@ -162,6 +181,58 @@ wasmedge_can_handle_container (libcrun_container_t *container, libcrun_error_t *
   return wasm_can_handle_container (container, err);
 }
 
+static int
+libwasmedge_configure_container (void *cookie, enum handler_configure_phase phase,
+                                 libcrun_context_t *context, libcrun_container_t *container,
+                                 const char *rootfs, libcrun_error_t *err)
+{
+  int ret;
+  cleanup_close int rootfsfd_cleanup = -1;
+  runtime_spec_schema_config_schema *def = container->container_def;
+
+  // Open the rootfs directory
+  if (rootfs != NULL)
+    {
+      rootfsfd_cleanup = open (rootfs, O_PATH | O_CLOEXEC);
+      if (UNLIKELY (rootfsfd_cleanup < 0))
+        return crun_make_error (err, errno, "open `%s`", rootfs);
+    }
+
+  if (phase != HANDLER_CONFIGURE_AFTER_MOUNTS)
+    return 0;
+
+  // Check if /usr/lib/wasmedge is already present in spec
+  if (def->linux && def->mounts)
+    {
+      for (size_t i = 0; i < def->mounts_len; i++)
+        {
+          if (strcmp (def->mounts[i]->destination, "/usr/lib/wasmedge") == 0)
+            return 0;
+        }
+    }
+
+  // Get the plugin folder path from the environment variable
+  char *plugin_mount_env = getenv ("WASMEDGE_PLUGIN_MOUNT");
+  printf ("RECEIVED WASMEDGE_PLUGIN_MOUNT value: %s\n", plugin_mount_env);
```

Comment:
Is this a debug statement or do we expect this to show in upstream run as well.

Reply:
We don't need that line. Fixed!